src/main/java/jenkins/branch/CustomOrganizationFolderDescriptor.java

```diff
@@ -70,6 +70,39 @@ public String getDisplayName() {
         return delegate.getDisplayName();
     }
 
+    /**
+     * Needed if it wants CustomOrganizationFolderDescriptor implementations are categorized in Jenkins 2.x.
+     *
+     * TODO: Override when the baseline is upgraded to 2.x
+     *
+     * @return A string with the Item description.
+     */
+    public String getDescription() {
+        return delegate.getDescription();
+    }
+
+    /**
+     * Needed if it wants CustomOrganizationFolderDescriptor implementations are categorized in Jenkins 2.x.
+     *
+     * TODO: Override when the baseline is upgraded to 2.x
+     *
+     * @return A string it represents a ItemCategory identifier.
+     */
+    public String getCategoryId() {
+        return delegate.getCategoryId();
```

Comment:
And would we not just want to return `nested-projects` by default, since by definition these will all fall into that category?